tests/nnapi/nnapi_gtest.skip.aarch64-linux.cpu

```diff
@@ -167,13 +167,22 @@ GeneratedTests.space_to_depth_float_2
 GeneratedTests.space_to_depth_float_3
 GeneratedTests.space_to_depth_quant8_1
 GeneratedTests.space_to_depth_quant8_2
-GeneratedTests.split_4D_quant8_nnfw
-GeneratedTests.split_quant8_1
+GeneratedTests.split_1D_int32_nnfw
+GeneratedTests.split_4D_int32_1_nnfw
+GeneratedTests.split_4D_int32_2_nnfw
+GeneratedTests.split_4D_int32_3_nnfw
+GeneratedTests.split_4D_int32_4_nnfw
+GeneratedTests.split_4D_int32_5_nnfw
+GeneratedTests.split_int32_1
+GeneratedTests.split_int32_1_relaxed
+GeneratedTests.split_int32_2
+GeneratedTests.split_int32_2_relaxed
+GeneratedTests.split_int32_3
+GeneratedTests.split_int32_3_relaxed
+GeneratedTests.split_int32_4
+GeneratedTests.split_int32_4_relaxed
```

Comment:
@dr-venkman Could you explain us why `int32` type `SPLIT` tests are added in skiplist?

Reply:
@hseok-oh , I was resolving conflicts, and must have accidentally added them in from the previous version. I will revert the changes. Thanks for spotting.